packages/sdk-bundles-api/src/replay-orchestrator/sdk-to-bundle/execute-replay.ts

```diff
@@ -178,6 +178,20 @@ export interface ReplayExecutionOptions {
    * This flag is useful to reduce noise when debugging.
    */
   essentialFeaturesOnly: boolean;
+
+  vercel?: {
+    deploymentProtectionBypassToken: string | null;
+  };
```

Comment:
Decided not to do a generic 'additionalHeaders' option, since we'll only want to send this header to origins we know are safe to send it to.